assests/javascript/game.js

```diff
@@ -0,0 +1,111 @@
+var rbSingers = ["usher","musiq soulchild","daniel caesar","chris brown","ella mai","her"];
+var wins = 0;
+var losses = 0;
+var guessesLeft = 10;
+var inserts = 0;
+var guessedLetter = [];
+var individual = [];
+var slots = [];
+var randomSing = "";
+
+var directionsText = document.getElementById("directions-text");
+
+gameStart()
+
+// COUNTERS
+
+document.getElementById("wins").textContent = wins;
+document.getElementById("losses").textContent = losses;
+document.getElementById("guessedLetter").textContent = guessedLetter.join;
+document.getElementById("guessesLeft").textContent = guessesLeft;
+
+// KEY PRESS
+
+document.onkeypress = function (event) {
+
+    directionsText.textContent = "";
+
+    letterCheck (event.key);
+}
+
+// FUNCTIONS
+
+function gameStart () {
+
+    randomSing = rbSingers[Math.floor(Math.random() *rbSingers.length)];
+
+    var individual = randomSing.split("");
+
+    var inserts = individual.length;
+
+    for (var i = 0; i < inserts; i++) {
+
+        slots.push("_");
+    }
+
+    document.getElementById("guessingWord").textContent = "  " + slots.join("  ");
+};
+
+// LETTER CHECK (NOT WORKING)
+
+function letterCheck (letter) {
+
+    var foundLetter = false;
+
+    for (var i = 0; i < inserts; i++) {
+
+        if (randomSing[i] === letter) {
+
+            foundLetter = true;
+            
+        }
+    }
+
+    if (foundLetter) {
```

Comment:
The way that your vars are set up it is always defining foundletter as false.  I would move some things around to get it to work.